### PR TITLE
fix docstrings (as well as some within interactive examples)

### PR DESCRIPTION
**Pull request by weikang9009**

### CODE REVIEW

File: spglm/glm.py [hunk 1]
```diff
@@ -245,28 +246,6 @@ class GLMResults(LikelihoodModelResults):
         normalized_cov_params   : array
                                 k*k, approximates [X.T*X]-1
 
-    Examples
```
Comment (TaylorOshan) [inline spglm/glm.py]: This example was deleted on purpose?

**Comment by TaylorOshan** [comment]
Any reason the docstring example for GLMResults is being deleted?

File: spglm/glm.py (hunk 1 above)
Comment (weikang9009) [reply spglm/glm.py]: Yes. Line 265 will run into an error. My understanding is that result1 (GLMResults instance) does not have attribute `predy`.

Comment (weikang9009) [reply spglm/glm.py]: I could be wrong. Feel free to add it back.

**Pull request merged**